storage/innobase/row/row0import.cc

```diff
@@ -1213,6 +1213,18 @@ row_import::match_index_columns(
 
 			err = DB_ERROR;
 		}
+
+		if (cfg_field->descending != field->descending) {
+			ib_errf(thd, IB_LOG_LEVEL_ERROR,
+				ER_TABLE_SCHEMA_MISMATCH,
+				"Index %s field %s is %s which does "
+				"not match wth .cfg file",
+				index->name(), field->name(),
+				field->descending
+				  ? "descending"
+				  : "ascending");
```

Comment:
I would remove the misspelled word "with" and use the SQL keywords `DESC` and `ASC`.